libs/ui/src/lib/notice/notice.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="cvi-notice__container cvi-notice--severity-{{ severity }}">
+  <div class="cvi-notice__icon-container">
+    <cvi-ng-icon [name]="getIconName()" [height]="24"></cvi-ng-icon>
+  </div>
+  <div class="cvi-notice__content">
+    <ng-container *ngTemplateOutlet="contentPrefix"></ng-container>
+    <ng-template #contentPrefix>
```

Comment:
We discussed this a bit and it might a recommended approach to replace the templates with content projection slots (`ng-content`). It is the default mechanism for slotting in Angular, so perhaps it should be preferred unless there's a specific reason not to.